Makefile

```diff
@@ -19,21 +19,44 @@ ifneq ($(MONO_OPTIONS),)
 export MONO_OPTIONS
 endif
 
+prefix=/usr/local
+
+
 include build-tools/scripts/msbuild.mk
 all::
 	$(MSBUILD) $(MSBUILD_FLAGS) $(SOLUTION)
 
 all-tests::
 	MSBUILD="$(MSBUILD)" tools/scripts/xabuild $(MSBUILD_FLAGS) Xamarin.Android-Tests.sln
-	
+
+install::
+	@if [ ! -d "bin/$(CONFIGURATION)" ]; then \
+		echo "run 'make all' before you execute 'make install'!"; \
+			exit 1; \
+	fi
+	-mkdir -p "$(prefix)/lib/mono/xbuild-frameworks"
+	-mkdir -p "$(prefix)/lib/xamarin.android"
+	-mkdir -p "/usr/lib/mono/xbuild/Xamarin/"
```

Comment:
This shouldn't assume `/usr/lib/mono`; it should either assume `$(prefix)/lib/mono/...`, *or* it should ["probe" for mono's install prefix](https://github.com/xamarin/xamarin-android/blob/91e0ba7/tools/scripts/xabuild#L135-L147).